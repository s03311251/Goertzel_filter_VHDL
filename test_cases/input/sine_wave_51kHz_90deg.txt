3FFF
3E5E
39A6
3251
291E
1EFE
14F8
0C12
0533
010E
0010
0252
079A
0F5D
18D2
2302
2CE4
3575
3BD8
3F64
3FBE
3CDD
370C
2EE2
2534
1AFE
114B
0917
0338
0047
0091
040E
0A64
12EC
1CC9
26FA
3076
3844
3D99
3FEB
3EFE
3AE8
3415
2B37
2134
1712
0DD9
0678
01B1
0000
0191
063B
0D84
16AF
20CD
2AD6
33C5
3AB0
3EE3
3FF2
3DC0
3886
30CD
275F
1D30
134A
0AB0
0441
00A5
003A
030C
08D0
10F0
1A99
24CE
2E86
36C4
3CB0
3FB1
3F77
3C0A
35C1
2D42
2369
1937
0FB6
07DD
0279
0017
00F5
04FB
0BC2
1498
1E97
28BB
31FC
3967
3E3D
3FFE
3E7E
