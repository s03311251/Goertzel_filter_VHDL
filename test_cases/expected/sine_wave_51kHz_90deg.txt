000619A0
FFFFE300
